src/app/public/modules/date-range-picker/date-range.service.ts

```diff
@@ -33,9 +33,6 @@ import {
   SKY_DEFAULT_CALCULATOR_CONFIGS
 } from './types/date-range-default-calculator-configs';
 
-/**
- * @internal
```

Comment:
Should we include a short description here for `SkyDateRangeService` similar to what we have above for `SkyDateRangePickerComponent`? Not sure if we need it, but just checking to see if we should have the description for consistency and/or to explain how the component and service work together.